plugins/agent-utilities/skills/fusion-parametric-design/tests/test_reconstruction_benchmark.py

```
@@ -0,0 +1,382 @@
+        self.assertAlmostEqual(
+            body["area_mm2"],
+            self.record["total_area"],
+            delta=STEP_AREA_REL_TOLERANCE * body["area_mm2"],
+        )
+        self.assertAlmostEqual(
+            body["volume_mm3"],
+            self.comparison["mesh_volume_mm3"],
+            delta=STEP_VOLUME_REL_TOLERANCE * body["volume_mm3"],
+        )
+        step_extent = [
+            high - low for low, high in zip(body["bbox_min_mm"], body["bbox_max_mm"])
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)